carapace-server/src/main/java/org/carapaceproxy/server/config/ActionConfiguration.java

```diff
@@ -63,9 +67,20 @@ public String getType() {
         return type;
     }
 
+    public List<String> getHeaders() {
+        return headers;
+    }
+
+    public ActionConfiguration addHeader(String id) {
+        if (!headers.contains(id)) {
```

Comment:
shouldn't this be treated as a configuration error ?

Reply:
I treaded it just like DirectorConfiguration#addBackend